frontend/public/components/service.jsx

```diff
@@ -206,6 +206,13 @@ const Details = ({ obj: s }) => {
         <div className="col-md-6">
           <SectionHeading text={t('public~Service routing')} />
           <dl>
+            <dt>{t('public~Hostname')}</dt>
+            <dd className="service-hostname">
+              <p>
```

Comment:
Do you need the `margin-bottom` style if these are `div`s instead of `p`?

Reply:
So I wanted to have similar styling (space between hostname and "Accessible within the cluster only") as below

Reply:
OK, if we need a custom style, please use a BEM classname.